gluten-ut/spark33/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -961,7 +961,7 @@ class VeloxTestSettings extends BackendTestSettings {
     // requires resource files from Vanilla spark jar
     .exclude("SPARK-32908: maximum target error in percentile_approx")
   enableSuite[GlutenCachedTableSuite]
-    .exclude("InMemoryRelation statistics")
+    .exclude("GLUTEN - InMemoryRelation statistics")
```

Comment:
Suppose we still need this test. You may need to enable columnar table cache in `GlutenCachedTableSuite`.

```
  override def sparkConf: SparkConf = {
    super.sparkConf.set("spark.sql.shuffle.partitions", "5")
    super.sparkConf.set(GlutenConfig.COLUMNAR_TABLE_CACHE_ENABLED.key, "true")
  }
```

Reply:
sure